src/App.css

```diff
@@ -33,6 +32,44 @@
   color: #61dafb;
 }
 
+.Background-logo {
+  background-color: darkred;
+  background-image: url('./Background.jpg');
+  background-repeat: no-repeat;
```

Comment:
Do we need all those CSS rules here?

Reply:
Yes, we do.

Reply:
I think we could avoid using background-repeat since we use 'cover' prop for bg size